src/sentry/relay/config/__init__.py

```diff
@@ -162,6 +164,40 @@ def get_dynamic_sampling_config(project: Project) -> Optional[Mapping[str, Any]]
     return None
 
 
+class TransactionNameRule(TypedDict):
+    pattern: str
+    expiry: str
+    scope: Mapping[str, str]
+    redaction: Mapping[str, str]
+
+
+def get_transaction_names_config(project: Project) -> Optional[Sequence[TransactionNameRule]]:
+    if not features.has("organizations:transaction_name_sanitization", project.organization):
+        return None
+
+    # TODO: swap the lines below when the format updates from Mapping to Sequence
+    # cluster_rules = get_rules(project)
+    cluster_rules: Sequence[Tuple[str, int]] = project.get_option(
+        "sentry:transaction_name_cluster_rules", {}
+    )
```

Comment:
Just merged https://github.com/getsentry/sentry/pull/42372, so you should now be able to call `get_sorted_rules`:

https://github.com/getsentry/sentry/blob/875d18c4cdd129db9f3231eb467f3e55f4d92e0b/src/sentry/ingest/transaction_clusterer/rules.py#L114